uwblueprint/starter-code-v2
# Add Simple Entity TypeScript API and frontend

## Notion ticket link
Apologies for the diff, majority of it is just copied from existing entity files


## Implementation description
* Created copies of existing entity API and frontend, without file storage



## Steps to test
1. Use the scrubber locally (yarn dev -t in starter-code-cli) with a TypeScript backend (and any API or database).
2. Login on http://localhost:3000 and verify that the new Create Simple Entity, Update Simple Entity, and Display Simple Entities pages work as expected.
3. Verify that the existing entity API is unaffected.


## What should reviewers focus on?
*


## Checklist
